sdk/communication/communication-identity/src/generated/src/models/mappers.ts

```diff
@@ -139,7 +139,7 @@ export const CommunicationError: coreHttp.CompositeMapper = {
         }
       },
       innerError: {
-        serializedName: "innerError",
+        serializedName: "innererror",
```

Comment:
@DominikMe we saw a similar error on the .net platform when performing this exercise IIRC

Reply:
This looks like the right final code. Once we update the changelog we can call it out as a bug fix